### PR TITLE
[ES] Add `History of osu! wiki`

### PR DESCRIPTION
**Pull request by GNosii**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/History_of_osu!/osu!_wiki/es.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Historia de la osu! wiki
+
+## MediaWiki (2011 - 2017)
+
+![](img/ow_MW.jpg "Captura de pantalla completa de la osu! wiki funcionando en MediaWiki, tomada en el 2017-02-17")
```
Comment (agatemosu) [inline wiki/History_of_osu!/osu!_wiki/es.md:5]:
```suggestion
![](img/ow_MW.jpg "Captura de pantalla completa de la osu! wiki funcionando en MediaWiki, tomada en 17/2/2017")
```

File: wiki/History_of_osu!/osu!_wiki/es.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+| Fecha, Hora | Descripción |
+| :-- | :-- |
+| 2011-12-05T08:13Z | Primera edición a la osu! wiki por Ephemeral. |
```
Comment (agatemosu) [inline wiki/History_of_osu!/osu!_wiki/es.md:9]:
```suggestion
| 5/12/2011 T08:13Z | Primera edición de la osu! wiki por Ephemeral. |
```

File: wiki/History_of_osu!/osu!_wiki/es.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+| 2011-12-06 | [La osu! wiki ya está abierta al público](https://osu.ppy.sh/community/forums/topics/68525). |
+| 2012-11 | La osu! wiki ahora tiene 3 administradores (Dellirium, Repflez). Antes, solo peppy era administrador. |
+| 2013-01-27T06:12Z | peppy integró la osu! wiki con el diseño del sitio web oficial. [Enlace a la publicación de peppy](https://osu.ppy.sh/community/forums/posts/2082803). |
```
Comment (agatemosu) [inline wiki/History_of_osu!/osu!_wiki/es.md:12]:
```suggestion
| 27/1/2013 T06:12Z | peppy integró la osu! wiki con el diseño del sitio web oficial. [Enlace a la publicación de peppy](https://osu.ppy.sh/community/forums/posts/2082803). |
```

File: wiki/History_of_osu!/osu!_wiki/es.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+| 2016-08-30 | La osu! wiki en MediaWiki es dejada de lado. Esta versión funcionaría hasta que se porten todas las imágenes y contenidos a la nueva interfaz. |
+| 2017-06 | La osu! wiki en MediaWiki es declarada como obsoleta. Su enlace ahora envia a la nueva interfaz de osu-web usando los recursos de la osu! wiki creados en el repositorio de GitHub. [Una copia de seguridad temporal de esta página sin funciones de MediaWiki puede ser vista aquí](https://web.archive.org/web/20171115173938/https://osu.ppy.sh/old-wiki/Main_Page). |
+
+## Repositorio de GitHub (2016 - presente)
+
+| Fecha | Descripción |
+| :-- | :-- |
+| 2016-08-26 | [peppy creó el repositorio de la osu! wiki](https://github.com/ppy/osu-wiki/tree/3433cbeeda9303a470647cad1c338d43f4272a2e). craftu y Xyloo portaron el contenido de MediaWiki al repositorio en una semana. |
+| 2016-09 | La mayoría de las páginas, excepto las imágenes se movieron al repositorio, con errores de sintaxis sin resolver, debido a las diferencias entre Markdown y MediaWiki. |
+| 2017-01 | Soporte de la osu! wiki en osu! web desarrollado por nanaya. |
```
Comment (agatemosu) [inline wiki/History_of_osu!/osu!_wiki/es.md:26]:
```suggestion
| 1/2017 | Soporte de la osu! wiki en osu!web desarrollado por nanaya. |
```

**Review by agatemosu — COMMENTED** [review]
In addition to this, could you change the dates to D/M/YYYY?

**Review by GNosii — COMMENTED** [review]
commited your suggestions!

File: wiki/History_of_osu!/osu!_wiki/es.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
+# Historia de la osu! wiki
+
+## MediaWiki (2011 - 2017)
+
+![](img/ow_MW.jpg "Captura de pantalla completa de la osu! wiki funcionando en MediaWiki, tomada en 17/2/2017")
+
+| Fecha, Hora | Descripción |
+| :-- | :-- |
+| 5/12/2011 T08:13Z | Primera edición de la osu! wiki por Ephemeral. |
+| 2011-12-06 | [La osu! wiki ya está abierta al público](https://osu.ppy.sh/community/forums/topics/68525). |
```
Comment (agatemosu) [inline wiki/History_of_osu!/osu!_wiki/es.md:10]:
```suggestion
| 16/12/2011 | [La osu! wiki ya está abierta al público](https://osu.ppy.sh/community/forums/topics/68525). |
```

File: wiki/History_of_osu!/osu!_wiki/es.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+| 2012-11 | La osu! wiki ahora tiene 3 administradores (Dellirium, Repflez). Antes, solo peppy era administrador. |
```
Comment (agatemosu) [inline wiki/History_of_osu!/osu!_wiki/es.md:11]:
```suggestion
| 11/2012 | La osu! wiki ahora tiene 3 administradores (Dellirium, Repflez). Antes, solo peppy era administrador. |
```

File: wiki/History_of_osu!/osu!_wiki/es.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+| 27/1/2013 T06:12Z | peppy integró la osu! wiki con el diseño del sitio web oficial. [Enlace a la publicación de peppy](https://osu.ppy.sh/community/forums/posts/2082803). |
+| 2014 | La osu! wiki se convierte oficialmente en un centro de información, reemplazando las versiones web de [Equipo](/wiki/People/osu!_team) y [Reglas](/wiki/Rules). También, ahora la osu! wiki tiene 5 administradores (Loctav, deadbeat). |
+| 2015 | La osu! wiki fue traducida a otros idiomas. Shiro reemplazó a Dellirium como uno de los administradores de la wiki. |
+| 2016-02-22 | La osu! wiki ahora tiene 8 administradores (Ukami y Galkan). |
```
Comment (agatemosu) [inline wiki/History_of_osu!/osu!_wiki/es.md:15]:
```suggestion
| 22/2/2016 | La osu! wiki ahora tiene 8 administradores (Ukami y Galkan). |
```

File: wiki/History_of_osu!/osu!_wiki/es.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+| 2016-04-01 | Se unió otro administrador a la osu! wiki (Nathanael). |
```
Comment (agatemosu) [inline wiki/History_of_osu!/osu!_wiki/es.md:16]:
```suggestion
| 1/4/2016 | Se unió otro administrador a la osu! wiki (Nathanael). |
```

File: wiki/History_of_osu!/osu!_wiki/es.md [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+| 2016-08-30 | La osu! wiki en MediaWiki es dejada de lado. Esta versión funcionaría hasta que se porten todas las imágenes y contenidos a la nueva interfaz. |
```
Comment (agatemosu) [inline wiki/History_of_osu!/osu!_wiki/es.md:17]:
```suggestion
| 30/8/2016 | La osu! wiki en MediaWiki es dejada de lado. Esta versión funcionaría hasta que se porten todas las imágenes y contenidos a la nueva interfaz. |
```

File: wiki/History_of_osu!/osu!_wiki/es.md [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+| 2017-06 | La osu! wiki en MediaWiki es declarada como obsoleta. Su enlace ahora envia a la nueva interfaz de osu-web usando los recursos de la osu! wiki creados en el repositorio de GitHub. [Una copia de seguridad temporal de esta página sin funciones de MediaWiki puede ser vista aquí](https://web.archive.org/web/20171115173938/https://osu.ppy.sh/old-wiki/Main_Page). |
```
Comment (agatemosu) [inline wiki/History_of_osu!/osu!_wiki/es.md:18]:
```suggestion
| 6/2017 | La osu! wiki en MediaWiki es declarada como obsoleta. Su enlace ahora envia a la nueva interfaz de osu-web usando los recursos de la osu! wiki creados en el repositorio de GitHub. [Una copia de seguridad temporal de esta página sin funciones de MediaWiki puede ser vista aquí](https://web.archive.org/web/20171115173938/https://osu.ppy.sh/old-wiki/Main_Page). |
```

File: wiki/History_of_osu!/osu!_wiki/es.md [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+
+## Repositorio de GitHub (2016 - presente)
+
+| Fecha | Descripción |
+| :-- | :-- |
+| 2016-08-26 | [peppy creó el repositorio de la osu! wiki](https://github.com/ppy/osu-wiki/tree/3433cbeeda9303a470647cad1c338d43f4272a2e). craftu y Xyloo portaron el contenido de MediaWiki al repositorio en una semana. |
```
Comment (agatemosu) [inline wiki/History_of_osu!/osu!_wiki/es.md:24]:
```suggestion
| 26/8/2016 | [peppy creó el repositorio osu-wiki](https://github.com/ppy/osu-wiki/tree/3433cbeeda9303a470647cad1c338d43f4272a2e). craftu y Xyloo portaron el contenido de MediaWiki al repositorio en una semana. |
```

File: wiki/History_of_osu!/osu!_wiki/es.md [hunk 12]
```diff
@@ -0,0 +1,37 @@
...
+| 2016-09 | La mayoría de las páginas, excepto las imágenes se movieron al repositorio, con errores de sintaxis sin resolver, debido a las diferencias entre Markdown y MediaWiki. |
```
Comment (agatemosu) [inline wiki/History_of_osu!/osu!_wiki/es.md:25]:
```suggestion
| 9/2016 | La mayoría de las páginas, excepto las imágenes se movieron al repositorio, con errores de sintaxis sin resolver, debido a las diferencias entre Markdown y MediaWiki. |
```

File: wiki/History_of_osu!/osu!_wiki/es.md [hunk 13]
```diff
@@ -0,0 +1,37 @@
...
+| 1/2017 | Soporte de la osu! wiki en osu!web desarrollado por nanaya. |
+| 2017-01-05 | El [proyecto de osu! wiki es sugerido por Ephemeralis](https://github.com/ppy/osu-wiki/projects/1). |
```
Comment (agatemosu) [inline wiki/History_of_osu!/osu!_wiki/es.md:27]:
```suggestion
| 5/1/2017 | El [proyecto osu! wiki es sugerido por Ephemeralis](https://github.com/ppy/osu-wiki/projects/1). |
```

File: wiki/History_of_osu!/osu!_wiki/es.md [hunk 14]
```diff
@@ -0,0 +1,37 @@
...
+| 2017-01-26 | La osu! wiki es mencionada en el blog de peppy. |
```
Comment (agatemosu) [inline wiki/History_of_osu!/osu!_wiki/es.md:28]:
```suggestion
| 26/1/2017 | La osu! wiki es mencionada en el blog de peppy. |
```

File: wiki/History_of_osu!/osu!_wiki/es.md [hunk 15]
```diff
@@ -0,0 +1,37 @@
...
+| 2017-05-22 | El [archivo de osu!news ahora es almacenado en la osu! wiki (solo en formato de texto)](https://github.com/ppy/osu-wiki/pull/252). |
```
Comment (agatemosu) [inline wiki/History_of_osu!/osu!_wiki/es.md:29]:
```suggestion
| 22/5/2017 | El [archivo de osu!news ahora es almacenado en la osu! wiki (solo en formato de texto)](https://github.com/ppy/osu-wiki/pull/252). |
```

File: wiki/History_of_osu!/osu!_wiki/es.md [hunk 16]
```diff
@@ -0,0 +1,37 @@
...
+| 2017-05-30 | [La función de redirección es añadida a la osu! wiki](https://github.com/ppy/osu-web/pull/1144). |
```
Comment (agatemosu) [inline wiki/History_of_osu!/osu!_wiki/es.md:30]:
```suggestion
| 30/5/2017 | [La función de redirección es añadida a la osu! wiki](https://github.com/ppy/osu-web/pull/1144). |
```

File: wiki/History_of_osu!/osu!_wiki/es.md [hunk 17]
```diff
@@ -0,0 +1,37 @@
...
+| 2017-06 | TPGPL es promovido a acceso de escritura en el repositorio de la osu! wiki. Un servicio de búsqueda en osu-web es añadido para la osu! wiki |
```
Comment (agatemosu) [inline wiki/History_of_osu!/osu!_wiki/es.md:31]:
```suggestion
| 6/2017 | TPGPL es promovido a acceso de escritura en el repositorio de la osu! wiki. Un servicio de búsqueda en osu-web es añadido para la osu! wiki |
```

File: wiki/History_of_osu!/osu!_wiki/es.md [hunk 18]
```diff
@@ -0,0 +1,37 @@
...
+| 2018-02-07 | [La función de etiquetado de artículos es añadida a la osu! wiki](https://github.com/ppy/osu-web/pull/2331) para permitir mejores resultados de búsqueda. |
```
Comment (agatemosu) [inline wiki/History_of_osu!/osu!_wiki/es.md:32]:
```suggestion
| 7/2/2018 | [La función de etiquetado de artículos es añadida a la osu! wiki](https://github.com/ppy/osu-web/pull/2331) para permitir mejores resultados de búsqueda. |
```

File: wiki/History_of_osu!/osu!_wiki/es.md [hunk 19]
```diff
@@ -0,0 +1,37 @@
...
+| 2018–2021 | *No documentado* |
+| 2021-05-12 | Primera implementación de las [cajas de información](https://github.com/ppy/osu-web/pull/7546) que podrían mejorar la funcionalidad en los artículos. |
```
Comment (agatemosu) [inline wiki/History_of_osu!/osu!_wiki/es.md:34]:
```suggestion
| 12/5/2021 | Primera implementación de las [cajas de información](https://github.com/ppy/osu-web/pull/7546) que podrían mejorar la funcionalidad en los artículos. |
```

File: wiki/History_of_osu!/osu!_wiki/es.md [hunk 20]
```diff
@@ -0,0 +1,37 @@
...
+| 2021-06-01 | Se creó una [implementación básica](https://github.com/ppy/osu/pull/12950) de la osu! wiki en [osu!(lazer)](/wiki/Client/Release_stream/Lazer) (solo en inglés). |
```
Comment (agatemosu) [inline wiki/History_of_osu!/osu!_wiki/es.md:35]:
```suggestion
| 1/6/2021 | Se creó una [implementación básica](https://github.com/ppy/osu/pull/12950) de la osu! wiki en [osu!(lazer)](/wiki/Client/Release_stream/Lazer) (solo en inglés). |
```

File: wiki/History_of_osu!/osu!_wiki/es.md [hunk 21]
```diff
@@ -0,0 +1,37 @@
...
+| 2021-08-08 | Se implementó la función de [galerías](https://github.com/ppy/osu-web/pull/8126) Ahora, las imágenes de un artículo pueden ser visualizadas en una galería centralizada. |
```
Comment (agatemosu) [inline wiki/History_of_osu!/osu!_wiki/es.md:36]:
```suggestion
| 8/8/2021 | Se implementó la función de [galerías](https://github.com/ppy/osu-web/pull/8126) Ahora, las imágenes de un artículo pueden ser visualizadas en una galería centralizada. |
```

File: wiki/History_of_osu!/osu!_wiki/es.md [hunk 22]
```diff
@@ -0,0 +1,37 @@
...
+| 2021-08-12 | Se añadió el soporte de [notas al pie](https://github.com/ppy/osu-web/pull/8125). Esto permitió incluir enlaces a notas y referencias al añadirlas en una sección específica al final de cada artículo. |
```
Comment (agatemosu) [inline wiki/History_of_osu!/osu!_wiki/es.md:37]:
```suggestion
| 12/8/2021 | Se añadió el soporte de [notas al pie](https://github.com/ppy/osu-web/pull/8125). Esto permitió incluir enlaces a notas y referencias al añadirlas en una sección específica al final de cada artículo. |
```

File: wiki/History_of_osu!/osu!_wiki/es.md [hunk 23]
```diff
@@ -0,0 +1,37 @@
+# Historia de la osu! wiki
+
+## MediaWiki (2011 - 2017)
+
+![](img/ow_MW.jpg "Captura de pantalla completa de la osu! wiki funcionando en MediaWiki, tomada en 17/2/2017")
+
+| Fecha, Hora | Descripción |
+| :-- | :-- |
+| 5/12/2011 T08:13Z | Primera edición de la osu! wiki por Ephemeral. |
```
Comment (GNosii) [inline wiki/History_of_osu!/osu!_wiki/es.md:9]:
```suggestion
| 5/12/2011 | Primera edición de la osu! wiki por Ephemeral. |
```
I believe for readibility we should remove the timestamp, what do you think?

Comment (agatemosu) [reply wiki/History_of_osu!/osu!_wiki/es.md:9]: or change it by (08:13 UTC)

Comment (agatemosu) [inline wiki/History_of_osu!/osu!_wiki/es.md:9]:
```suggestion
| 5/12/2011 (08:13 UTC) | Primera edición de la osu! wiki por Ephemeral. |
```

File: wiki/History_of_osu!/osu!_wiki/es.md [hunk 24]
```diff
@@ -0,0 +1,37 @@
...
+| 16/12/2011 | [La osu! wiki ya está abierta al público](https://osu.ppy.sh/community/forums/topics/68525). |
+| 11/2012 | La osu! wiki ahora tiene 3 administradores (Dellirium, Repflez). Antes, solo peppy era administrador. |
+| 27/1/2013 T06:12Z | peppy integró la osu! wiki con el diseño del sitio web oficial. [Enlace a la publicación de peppy](https://osu.ppy.sh/community/forums/posts/2082803). |
```
Comment (agatemosu) [inline wiki/History_of_osu!/osu!_wiki/es.md:12]:
```suggestion
| 27/1/2013 (06:12 UTC) | peppy integró la osu! wiki con el diseño del sitio web oficial. [Enlace a la publicación de peppy](https://osu.ppy.sh/community/forums/posts/2082803). |
```

**Review by agatemosu — COMMENTED** [review]
TheLastCookie recommended putting it like this

File: wiki/History_of_osu!/osu!_wiki/es.md (hunk 23 above)
Comment (Walavouchey) [reply wiki/History_of_osu!/osu!_wiki/es.md:9]: this change can be made in the english article as well

**Review by agatemosu — DISMISSED** [review]

**Comment by GNosii** [comment]
I'll wait for #9184 to be merged before any new changes to this branch.

**Review by agatemosu — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**